src/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -66,6 +67,57 @@ public override IEnumerable<TimeFunction> TimeFunctions(bool requiresRoundtrippi
                 DateTimeKind.Utc);
         }
 
+        [Fact]
+        public void CopyToMillisecondPresent()
+        {
+            FileInfo input = new FileInfo(Path.Combine(TestDirectory, GetTestFileName()));
```

Comment:
This can use GetTestFilePath(): it does the Path.Combine of the TestDirectory with GetTestFileName().